project/AkkaBuild.scala

```diff
@@ -251,6 +253,14 @@ object AkkaBuild {
     mavenLocalResolverSettings,
     docLintingSettings,
     JdkOptions.targetJdkSettings,
+
+    // a workaround for https://github.com/akka/akka/issues/27661
+    // see also project/Protobuf.scala that introduces /../ to make "intellij happy"
+    MultiJvm / assembly / fullClasspath := {
+      val old = (MultiJvm / assembly / fullClasspath).value.toVector
+      val files = old.map(_.data.getCanonicalFile).distinct
+      files map { x => Attributed.blank(x) }
+    },
```

Comment:
This will help in conjunction with https://github.com/akka/akka/pull/28055
but specifically help as well given https://github.com/akka/akka/blob/master/project/Protobuf.scala#L32-L33